app.py

```diff
@@ -11,8 +12,13 @@ def index():
     if request.method == 'POST':
         NotamRequest = Models.NotamRequest(request.form)
 
-        # pass the form to getNOTAM to make API request
-        # apiOutput = GetNOTAM.getNOTAM(NotamRequest)
+        # get lat/long of airports
```

Comment:
is there an issue entered in Github for figuring out how to determine this?

Reply:
we wrote it on the board in the last meeting and I put it on the google doc notes for the meeting. But no not on github